WPMedia/fusion-news-theme-blocks
# THEMES-836: Quilted Image block | Resizer v2

## Description

This PR updates the quilted image block to utilize resizer v2.

## Jira Ticket

- [THEMES-836](https://arcpublishing.atlassian.net/browse/THEMES-836)

## Acceptance Criteria

- Quilted Image - Arc Block is using Resizer V2

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-836`
2. Checkout the `commerce` branch for the feature pack.
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/quilted-image-block`
4. Set up a test page and add the quilted image block.
5. Use the 'Replace image' buttons on the block in the preview area to set the images.
6. Fill out the other custom fields on the block, for the overlay and button text, and button URL.
7. The images should be using resizer v2 URLs.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
